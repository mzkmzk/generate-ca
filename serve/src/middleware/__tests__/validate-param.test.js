const validateParam = require('../validate-param')

it('测试参数校验', async function(){
  try {
    await validateParam({ query: { dns0: '' } }, null,  {
      rules: [ { key: 'dns0', descriptor: [ 'empty', 'dns'] } ]
    })
  } catch(e) {
    expect(e).toMatchSnapshot()
  }
})