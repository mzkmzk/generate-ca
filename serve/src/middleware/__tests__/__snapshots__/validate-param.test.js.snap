// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`测试参数校验 1`] = `[Error: dns0 不能为空]`;
